.circleci/config.yml

```diff
@@ -2,25 +2,28 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:8
+      - image: circleci/node:10
```

Comment:
Node 8 doesn't reach end of life till April, so I want to keep it for now -- my goal is to make sure stuff builds and works on the oldest version of node that's still supported